requirements_test_pre_commit.txt

```diff
@@ -0,0 +1,7 @@
+autoflake==1.4
+black==20.8b1
+flake8==3.9.0
+isort==5.8.0
+mypy==0.812
+pyupgrade==2.11.0
+black-disable-checker==1.0.0
```

Comment:
Currently, `astroid` only uses `pyupgrade` (in v2.12.0) and `black` of those here. Unless I'm missing something

Reply:
No you're right. It can't use pylint either because once you start analyzing everything there's a hundred of warnings.

Reply:
Are you sure with `pylint`? I just ran it locally and the `astroid` directory doesn't have any issues. That's also the only one tox checked.
```
commands = pylint -rn --rcfile={toxinidir}/pylintrc {toxinidir}/astroid
```

Reply:
Yeap pylint does not work on package without ``__init__.py``. So if you add an init in ``astroid/brain`` everything starts to be analysed. Or use pre-commit because it launch pylint file by file.

```yaml
  - repo: local
    hooks:
      - id: pylint
        name: pylint
        entry: pylint
        language: system
        types: [python]
        args:
          [
            "-rn",
            "-sn",
            "--rcfile=pylintrc",
          ]
        exclude: tests/testdata
```